### DISCUSSION TITLE
website-introduction/

### DISCUSSION THREAD

**Discussion by giscus** [RuanZhongNan/RPGMV-dev-notes#137] [Announcements]
# website-introduction/

网站说明 本网站将会重点专注于插件开发，源码解析，以及其他前沿技术在mv项目中的应用。 本网站的措辞倾向于专业的开发用语。不提供基础的编程常识介绍和JavaScript的基础学习。 本网站对没有编程基础的开发者并不友好，建议你先学好JavaScript+html的知识，再来阅读本网站。 本网站提供 基于 rmmv 的 js 基础 rmmv 源码解析 现...

https://www.ruan-cat.com/website-introduction/

**Comment by TheLushHill** [comment]
支持，发现了一个好网站😊
